### DISCUSSION TITLE
MPFS AMP mode

### DISCUSSION THREAD

**Discussion by gsriker77** [polarfire-soc/polarfire-soc-documentation#206] [Applications and demos] [answered]
In AMP mode, can we run 4 different Linux instances on the 4 U54 cores (one Linux instance per U54 core) ?

Thanks,

**Comment by TommyMurphyTM1234** [comment, marked as answer]
Doesn't seem so:

* https://github.com/polarfire-soc/polarfire-soc-documentation/blob/master/asymmetric-multiprocessing/amp.md#amp-on-polarfire-soc

> PolarFire SoC can be configured to run up to two independent software contexts. Each software context can have its own operating system, memory regions and hardware resources assigned.
> 
> The supported PolarFire SoC AMP software architecture is described below:
>
> * 1x E51 monitor core which is dedicated to running the Hart Software Services (HSS)
>
> * 4x U54 application cores which can be distributed between two independent software contexts
